pkg/scheduler/framework/runtime/framework.go

```diff
@@ -68,17 +63,26 @@ func (frw *frameworkImpl) RunFilterPlugins(ctx context.Context, placement *polic
 func (frw *frameworkImpl) RunScorePlugins(ctx context.Context, placement *policyv1alpha1.Placement, clusters []*clusterv1alpha1.Cluster) (framework.PluginToClusterScores, error) {
 	result := make(framework.PluginToClusterScores, len(frw.filterPlugins))
 	for _, p := range frw.scorePlugins {
-		for i, cluster := range clusters {
+		var scoreList framework.ClusterScoreList
+		for _, cluster := range clusters {
 			score, res := p.Score(ctx, placement, cluster)
 			if !res.IsSuccess() {
 				return nil, fmt.Errorf("plugin %q failed with: %w", p.Name(), res.AsError())
 			}
-			result[p.Name()][i] = framework.ClusterScore{
```

Comment:
Wow, just noticed that the `result[p.Name()]` is not initialized(nil slice), right?

Reply:
Right, this bug has not been triggered because the score plugin does not work😂.